sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1413,6 +1413,14 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val CONTINUOUS_STREAMING_EPOCH_BACKLOG_QUEUE_SIZE =
+    buildConf("spark.sql.streaming.continuous.epochBacklogQueueSize")
+      .internal()
```

Comment:
Why internal? Won't that hide it from users (e.g. if you use "set -v")?

Reply:
My idea here that Spark should work without user intervention and I can't really think any use-case where user need to touch this. On the other hand hardcoding things is also not classical good solution so decided to make it internal. If you think it's better to expose it just let me know, no hard opinions.

Reply:
Made it public.